docs/email/postfix/postfix-spf-dkim-debian-8.md

```diff
@@ -0,0 +1,380 @@
+
+7.  Make sure the ownership and permissions on `/etc/opendkim` and it's contents are correct by running the following commands:
+
+    cd /etc
+    chown -R opendkim:opendkim /etc/opendkim
+    chmod -R go-rw /etc/opendkim/keys
+
+8.  Check that OpenDKIM starts correctly by:
+
+    systemctl restart opendkim
+
+You should get no error messages from it. If you get errors, use:
+
+    systemctl status -l opendkim
+
+to get the status and untruncated error messages.
+
+### Setting up DNS
+
+As with SPF, DKIM uses TXT records to hold information about the signing key for each domain. Using YYYYMM as above, you need to make a TXT record for the host `YYYYMM._domainkey` for each domain you handle mail for. It's value can be found in the `example.txt` file for the domain. Those files look like this:
+
+{: .file}
+example.txt
+:   ~~~ text
+    201510._domainkey  IN  TXT ( **"v=DKIM1; k=rsa; s=email; "
+        "p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAu5oIUrFDWZK7F4thFxpZa2or6jBEX3cSL6b2TJdPkO5iNn9vHNXhNX31nOefN8FksX94YbLJ8NHcFPbaZTW8R2HthYxRaCyqodxlLHibg8aHdfa+bxKeiI/xABRuAM0WG0JEDSyakMFqIO40ghj/h7DUc/4OXNdeQhrKDTlgf2bd+FjpJ3bNAFcMYa3Oeju33b2Tp+PdtqIwXR"
+        "ZksfuXh7m30kuyavp3Uaso145DRBaJZA55lNxmHWMgMjO+YjNeuR6j4oQqyGwzPaVcSdOG8Js2mXt+J3Hr+nNmJGxZUUW4Uw5ws08wT9opRgSpn+ThX2d1AgQePpGrWOamC3PdcwIDAQAB"** )  ; ----- DKIM key 201510 for example.com
+
+    ~~~
+
+The value inside the parentheses (emphasized above) is what you want. Select the entire region from the double-quote before `v=DKIM1` on to the final double-quote before the closing parentheses. From the above file that would be:
+
+{: .file-excerpt}
+record.txt
+:   ~~~ text
+    "v=DKIM1; k=rsa; s=email; "
+    "p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAu5oIUrFDWZK7F4thFxpZa2or6jBEX3cSL6b2TJdPkO5iNn9vHNXhNX31nOefN8FksX94YbLJ8NHcFPbaZTW8R2HthYxRaCyqodxlLHibg8aHdfa+bxKeiI/xABRuAM0WG0JEDSyakMFqIO40ghj/h7DUc/4OXNdeQhrKDTlgf2bd+FjpJ3bNAFcMYa3Oeju33b2Tp+PdtqIwXR"
+    "ZksfuXh7m30kuyavp3Uaso145DRBaJZA55lNxmHWMgMjO+YjNeuR6j4oQqyGwzPaVcSdOG8Js2mXt+J3Hr+nNmJGxZUUW4Uw5ws08wT9opRgSpn+ThX2d1AgQePpGrWOamC3PdcwIDAQAB"
+    ~~~
+
+It's broken into chunks because of limitations in Bind (one of the most popular DNS server packages) and because of size limitations in the UDP protocol that's usually used for DNS requests and responses. Copy that region and paste it into the value for the TXT record.
+
+If you're using Linode's DNS manager, this is what the add TXT record screen will look like when you have it filled out:
+
+![Linode DNS manager add TXT record](/docs/assets/Postfix_DKIM_TXT_record.png)
+
+Repeat this for every domain you handle mail for, using the `.txt` file for that domain.
+
+### Test your configuration
+
+Test the keys for correct signing and verification using the `opendkim-testkey` command:
+
+    opendkim-testkey -d example.com -s YYYYMM
```

Comment:
As noted earlier (though not in line) This test fails for me:

```
opendkim-testkey -d mydomain.com -s 201512 -vv
opendkim-testkey: using default configfile /etc/opendkim.conf
opendkim-testkey: checking key '201512._domainkey.mydomain.com'
opendkim-testkey: syntax error in key data (ASCII 0x22 at offset 0)
```

Looking at this full output alongside my note on L273, I'm wondering if it's related to the name given to the txt record.